src/js/factual_background.js

```diff
@@ -26,16 +28,25 @@ class TabInfo {
     this.updateBrowserAction();
   }
 
-  setLoading() {
-    this.status = 'loading';
+  setLoading(url) {
```

Comment:
nie powinnismy tu ustawiac `contentLoaded` na `false`?

Reply:
Ja zrozumiałem zmienną `contentLoaded` jako to, czy w tej zakładce kiedykolwiek jakiś content był załadowany, a nie jako zmienną sprawdzającą czy jesteśmy między statusem 'loading', a 'completed'.. Nie widziałem, żeby status był gdziekolwiek wykorzystywany, więc usunąłem zapisywanie statusu. Jak ty to widzisz?

Reply:
Ok, możemy chyba równie dobrze operować na zmiennej `contentLoaded`. Może lepszą nazwą dla tej metody zamiast `setLoading` byłoby `setCurrentUrl`? W sumie updatujemy tutaj url a nie status załadowania.

Reply:
Masz rację, zmieniłem na `updateUrl`.